src/views/Pools/components/PoolCard/CardActions/HarvestActions.tsx

```diff
@@ -25,9 +24,16 @@ const HarvestActions: React.FC<HarvestActionsProps> = ({
   isLoading = false,
 }) => {
   const { t } = useTranslation()
-  const earningTokenPrice = useGetApiPrice(earningToken.address ? getAddress(earningToken.address) : '')
+
+  const earningTokenFarmForPriceCalc = useFarmFromTokenSymbol(earningToken.symbol)
+  const earningTokenPrice = useTokenPriceBusd(earningTokenFarmForPriceCalc.pid)
```

Comment:
`MIX` pool janks out here because it doesn't have an `earningTokenFarmForPriceCalc`. Previously I've just avoided rendering the entire component when this stuff that the hook requires isn't present, but the MIX pool _does_ need some of the stuff in this component.